tests/client.rs

```diff
@@ -134,7 +131,7 @@ async fn test_directory_not_existing() {
     let root_cause = error.root_cause();
     assert_eq!(
         format!("{}", root_cause),
-        "Corrupt cache: directory cache does not exist"
+        "Corrupt cache: cache-directory doesn't exist"
```

Comment:
if you insist on changing the message, you need to change this test as well: https://github.com/c4dt/lightarti-rest-android/blob/e34f0ee7e58f5ae962d88d6ef612f6ad04ebfd37/artiwrapper/src/androidTest/java/org/c4dt/artiwrapper/JniTest.java#L85

Reply:
Thanks - I thought of releasing a new version (should it even be a point version, like 0.5.0?) and then adjusting the android and iOS libraries.

https://github.com/c4dt/lightarti-rest-android/issues/32

Reply:
Oki :+1: also OK for the point version, since we moved functionality to here